visits-service-new/src/test/java/com/petclinic/visits/visitsservicenew/PresentationLayer/VisitsControllerIntegrationTest.java

```diff
@@ -214,7 +215,7 @@ void getVisitsForStatus(){
     }
 
 
-    @Test
+/*    @Test
```

Comment:
It was a test that was failing because of the new addition, I than created another test but did not want to delete the old test method so i just commented it/